submissions/fabunmiTolulope/html-forms/contact-form.html

```diff
@@ -13,8 +13,8 @@ <h1>Welcome to Greenprint Integrated Services</h1>
         <ul>
                 <li><a href="#"> Home</a></li>
                 <li><a href="#">Blog</a> </li>
-                <li><a href="../HTML/Productpage.html">Productpage</a> </li>
-                <li><a href="../html-forms/contact-form.html">Contact Us</a> </li>
+                <li><a href="#">Productpage</a> </li>
```

Comment:
Revert the change to how it was before. Add the links back

Reply:
I have added the navigation links

Reply:
![Screenshot_20201030-121839_Samsung Internet](https://user-images.githubusercontent.com/70230223/97699314-358b0d80-1aaa-11eb-8501-a676b50434b2.jpg)